webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -59,9 +66,14 @@ export default class CreateIssueModal extends PureComponent {
             e.preventDefault();
         }
 
+        const {post, channelId} = this.props;
+
+        const postId = (typeof post === 'undefined') ? '' : post.id;
```

Comment:
You don't need to do the typeof here because undefined is false.

Reply:
true true, I was just being really 'correct' (protecting against all the other falsey values... which will never happen here... :)